packages/sanity/src/core/studio/StudioLayout.tsx

```diff
@@ -110,9 +117,11 @@ export function StudioLayout() {
   )
 }
 
+// @TODO re-use `LoadingComponent`, which is `LoadingScreen` by default, to reduce "popping" during initial load
```

Comment:
Why in the future and not now? It would require more refactoring and rewriting?

Reply:
Out of scope, don't have the time as our deadline is rapidly approaching 💀